index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tic Tac Toe</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav>
+        <ul>
+            <li>MyTicToe</li>
+        </ul>
+    </nav>
+    <main>
+        <div class="game-container">
+            <div class="box">
+                <span class="text">
+                    <!-- here the entries of X or 0 will be displayed -->
+                </span>
+            </div>
+            <div class="box">
+                <span class="text">
+                    <!-- here the entries of X or 0 will be displayed -->
+                </span>
+            </div>
+            <div class="box">
+                <span class="text">
+                    <!-- here the entries of X or 0 will be displayed -->
+                </span>
+            </div>
+            <div class="box">
+                <span class="text">
+                    <!-- here the entries of X or 0 will be displayed -->
+                </span>
+            </div>
+            <div class="box">
+                <span class="text">
+                    <!-- here the entries of X or 0 will be displayed -->
+                </span>
+            </div>
+            <div class="box">
+                <span class="text">
+                    <!-- here the entries of X or 0 will be displayed -->
+                </span>
+            </div>
+            <div class="box">
+                <span class="text">
+                    <!-- here the entries of X or 0 will be displayed -->
+                </span>
+            </div>
+            <div class="box">
+                <span class="text">
+                    <!-- here the entries of X or 0 will be displayed -->
+                </span>
+            </div>
+            <div class="box">
+                <span class="text">
+                    <!-- here the entries of X or 0 will be displayed -->
+                </span>
+            </div>
+        </div>
+        <div class="game-info">
+            <h1>Welcome to Tic Tac Toe Game</h1>
+            <div class="attributes">
+                <p class="info">Turn for Player 1</p>
+                <button id="reset">Reset</button>
+            </div>
+        </div>
+    </main>
+    <script src="script.js"></script>
```

Comment:
scripts should be in head section. use defer attribute if the scripts need to access dom elements